test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -514,7 +521,7 @@ private static void testCheckCastOnInvalidQtype(ClassWriter cw) {
 		MethodVisitor mv = cw.visitMethod(ACC_PUBLIC + ACC_STATIC, "testCheckCastOnInvalidQtype", "()Ljava/lang/Object;", null, null);
 		mv.visitCode();
 		mv.visitInsn(ACONST_NULL);
-		mv.visitTypeInsn(CHECKCAST, "QClassDoesNotExist;");
+		mv.visitTypeInsn(CHECKCAST, "LClassDoesNotExist;");
```

Comment:
As Q type is being replaced by L type, the name of this test `testCheckCastOnInvalidQtype` can be updated as well.

Reply:
Actually I think this test can be removed entirely, testCheckCastOnInvalidLtype already exists.